dashboard/app/models/lesson.rb

```diff
@@ -291,6 +291,17 @@ def summarize(include_bonus_levels = false, for_edit: false)
     lesson_summary.freeze
   end
 
+  def summarize_for_calendar
+    {
+      id: id,
+      title: localized_title,
+      duration: lesson_activities.map(&:summarize).sum {|activity| activity[:duration] || 0},
+      assessment: !!assessment,
+      unplugged: script.is_migrated ? unplugged : display_as_unplugged, # TODO: Update to use unplugged property
+      url: lesson_path(id: id)
```

Comment:
Locally, clicking on a lesson is directing me to the first level in a lesson rather than the lesson plan. Is that expected?

Reply:
I recently updated the lesson_paths so this will probably need to be updated to 

```
url: script_lesson_path(script, self)
```

Reply:
weird, that's not expected. Which unit are you testing it on locally? CSP Unit 1 ('21-'22) is directing me to the lesson plan

Reply:
You might need to pull the most recent staging.

Reply:
oh yup, you're totally right. I just fixed it :)